File: CSharp/2395.cs
using System;
using System.Linq;

class URI {

    static void Main(string[] args) 
    { 
        int print=0;
        int[] conteiner = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int[] navio =  Console.ReadLine().Split().Select(int.Parse).ToArray();

        if(conteiner[0]>navio[0] || conteiner[1]>navio[1] || conteiner[2]>navio[2])
        {print=0;}
        else
        {
            print = (navio[0]/conteiner[0])*(navio[1]/conteiner[1])*(navio[2]/conteiner[2]);
        }
        Console.WriteLine(print);
        

    }

}

File: Python/2395.py
a =list(map(int, input().split()))
b= list(map(int, input().split()))

if a[0]>b[0] or a[1]>b[1] or a[2]>b[2]:
    qtd=0
else:
    qtd= (b[0]//a[0])*(b[1]//a[1])*(b[2]//a[2])
print(qtd)